Inputs/Input_Info.py

```diff
@@ -11,9 +11,9 @@
 INPUT_m4l_high = 160.0
 
 # Lumi for each year
-Lumi_2016 = 36.6
-Lumi_2017 = 41.5
-Lumi_2018 = 59.7
+Lumi_2016 = 36.31
```

Comment:
2016 the lumi should be 36.33.

19.52 + 16.81 = 36.33

https://docs.google.com/presentation/d/1gEF2jExqS1hWs-dlYgxrTT2xQztYbBZcrLdw45Aiy7g/edit#slide=id.ge8cc301637_0_55